config/core/configmaps/sugar.yaml

```diff
@@ -0,0 +1,46 @@
+# Copyright 2022 The Knative Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: config-sugar
+  namespace: knative-eventing
+  labels:
+    eventing.knative.dev/release: devel
+    app.kubernetes.io/version: devel
+    app.kubernetes.io/name: knative-eventing
+data:
```

Comment:
can you move this under `_example:` and provide a checksum. See https://github.com/knative/eventing/blob/main/config/core/configmaps/observability.yaml for an example. 

the checksums are generated by this script: https://github.com/knative/eventing/blob/main/hack/update-checksums.sh

Reply:
Thanks, updated